src/Microsoft.Tye.Core/Serialization/YamlSerializer.cs

```diff
@@ -17,5 +18,19 @@ public static ISerializer CreateSerializer()
                     .WithEmissionPhaseObjectGraphVisitor(args => new OmitDefaultAndEmptyArrayObjectGraphVisitor(args.InnerVisitor))
                     .Build();
         }
+
+        public static string ConvertToJson(string yamlContent)
+        {
+            var deserializer = new DeserializerBuilder().Build();
+            var yamlObject = deserializer.Deserialize<string>(yamlContent);
+
+            var serializer = new SerializerBuilder()
+                .JsonCompatible()
+                .Build();
+
+            var json = serializer.Serialize(yamlObject);
+
+            return json;

```

Comment:
It would be really nice if this could also inject `$schema` pointing at: https://raw.githubusercontent.com/dotnet/tye/master/src/schema/tye-schema.json

Reply:
added.